cc/toolchains/args/BUILD

```diff
@@ -23,6 +23,7 @@ cc_feature_set(
         "//cc/toolchains/args/preprocessor_defines:feature",
         "//cc/toolchains/args/runtime_library_search_directories:feature",
         "//cc/toolchains/args/shared_flag:feature",
+        "//cc/toolchains/args/strip_flags:feature",
```

Comment:
note that this adds a feature, but the legacy toolchain setup doesn't have a feature for this, it's just enabled straight on the action config, shouldn't hurt either way